requirements/test.txt

```diff
@@ -1,14 +1,14 @@
 # Requirements for test runs
 
 codecov==2.0.15
-diff-cover==1.0.4
+diff-cover==3.0.1
 django-nose==1.4.4
-lazy==1.1
-pytest-cov==2.6.0
-tox==3.1.3
-tox-battery==0.5.1
+lazy==1.4
+pytest-cov==2.10.0
+tox==3.16.1
+tox-battery==0.6.1
 mock==3.0.5
 
 # Github requirements
-git+https://github.com/edx/django-pyfs.git@1.0.3#egg=django-pyfs==1.0.3
+git+https://github.com/edx/django-pyfs.git@2.2#egg=django-pyfs==2.2
 git+https://github.com/edx/xblock-sdk.git#egg=xblock-sdk
```

Comment:
@taniwha, `xblock-sdk` can now be installed from PyPi. See [this](https://github.com/edx/xblock-sdk/releases/tag/0.2.0).